config/settings.yml

```diff
@@ -93,6 +97,9 @@ remote_api_url: <%= (ENV['GRAPE_SWAGGER_ROOT_URL'] || 'http://localhost:3001') +
 
 # contact email address
 laa_contact_email: crowncourtdefence@legalaid.gsi.gov.uk
+advocates_email: advocates-fee@legalaid.gsi.gov.uk
```

Comment:
Moved some static settings to configurations because they were set in a view ( 😱  ) until now.

This also makes it re-usable across the app.